tests/fixture/tmpnet/node_process.go

```diff
@@ -112,7 +112,22 @@ func (p *NodeProcess) Start(w io.Writer) error {
 		return fmt.Errorf("failed to remove stale process context file: %w", err)
 	}
 
+	// All argument are provided in the flags file
 	cmd := exec.Command(p.node.RuntimeConfig.AvalancheGoPath, "--config-file", p.node.getFlagsPath()) // #nosec G204
+
+	// Ensure process is detached from the parent process so that an error in the parent will not affect the child
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		Setsid: true,
+	}
+
+	// Redirect stdout and stderr error to a file since it will detach from the parent process
+	outfile, err := os.OpenFile("nohup.out", os.O_RDWR|os.O_CREATE|os.O_TRUNC, perms.ReadWrite)
+	if err != nil {
+		return fmt.Errorf("failed to create stderr/stdout output file: %w", err)
+	}
+	cmd.Stdout = outfile
+	cmd.Stderr = outfile
```

Comment:
This seems slightly weird.
- Are multiple nodes going to be writing to the same file?
- Should this just drop the output rather than writing it to a file?
- Where is this file going to be placed?
  - Should it be placed in the node's folder?